src/code_actions.zig

```diff
@@ -284,6 +341,10 @@ const DiagnosticKind = union(enum) {
             return DiagnosticKind{
                 .omit_discard = parseEnum(DiscardCat, msg["discard of ".len..]) orelse return null,
             };
+        } else if (std.mem.startsWith(u8, msg, "Functions should")) {
```

Comment:
i thinks its better if you check for the entire message `Functions should be camelCase`